src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -154,6 +154,66 @@ static int pcm_convert_s32_to_alaw(const struct audio_stream *source,
 }
 #endif /* CONFIG_PCM_CONVERTER_FORMAT_A_LAW && CONFIG_PCM_CONVERTER_FORMAT_S32LE */
 
+#if CONFIG_PCM_CONVERTER_FORMAT_MU_LAW && CONFIG_PCM_CONVERTER_FORMAT_S32LE
+static int pcm_convert_mulaw_to_s32(const struct audio_stream *source,
+				    uint32_t ioffset, struct audio_stream *sink,
+				    uint32_t ooffset, uint32_t samples, uint32_t chmap)
+{
+	uint8_t *src = audio_stream_get_rptr(source);
```

Comment:
in general - source pointers probably can be `const`? Might also give the compiler a clue for some slight optimisations here and there. Would be good to use everywhere